### ISSUE TITLE
Announce zoom level and visible features after zooming via VoiceOver

### ISSUE THREAD

**Issue by 1ec5** [mapbox/mapbox-gl-native#4821]
After zooming in or out using VoiceOver’s upward or downward swipe gestures, MGLMapView should announce the new zoom level and summarize the visible features. For comparison, MapKit provides a readout similar to the following:

> Zoom 15x. Places visible: AutoZone, McDonald’s. 16 roads visible.

Announcing the zoom level would be easy, but I don’t know how to manually trigger a readout that doesn’t correspond to an element on-screen. Summarizing the visible roads or points of interest would require #352. In the meantime, we’d want to summarize the visible annotations, as the fact that they’re visible is perhaps more intentional than the fact that some road’s label happens to be visible.

/ref #1496
/cc @friedbunny

**Comment by 1ec5** [comment]
An implementation is ready in #9950.
